rpc/cloudagent/cloud/etcdcluster/moc_cloudagent_etcdcluster.proto

```diff
@@ -0,0 +1,36 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the Apache v2.0 license.
```

Comment:
If you would like to expose etcd as a service in cloudagent, then the hierarchy has to be built like below.

cloud/etcdcluster
cloud/etcdcluster/peers

Look at Keyvault and other child services under KeyVault

The use case would be users can use cloud agent to create etcd cluster, is what you can get with exposing this 
Like we have kubernetes, etcd would be another service, which can be used.

Reply:
Just to make sure I understand, did you mean ... and other child services under Group? There do not seem to be child services under KeyVault.

Reply:
Ok, took a bit, but I think I have decent idea about the structure of the code and have implemented: etcdserver and etcdcluster are registred under location, and etcdserver is also registered under etcdcluster. One of these days, I would like to learn how did you guys come up with this design. Need to clean it up a bit, will update PR on Monday.